modules/standard/BigInteger.chpl

```diff
@@ -626,6 +626,25 @@ module BigInteger {
     return ret:t;
   }
 
+  @chpldoc.nodoc
+  inline operator :(const ref x: bigint, type t: string) {
```

Comment:
Is this necessary for compatibility with some program(s)? Should it have a deprecation?

Reply:
Casting bigint to string is used in several tests, and was previously relying on the universal cast to string, so I added this implementation to keep those working.

I'm glad you pointed this out though because I realized the underlying `get_str` method is already handling locality checks, so I removed that logic from the implementation.